features/step_definitions/move_to_trash_step.rb

```diff
@@ -0,0 +1,5 @@
+Then(/^"([^"]*)" should have "([^"]*)" messages$/) do |user, expected_count|
+  @receiver = User.find_by(name: user.downcase)
+  count = @receiver.mailbox.inbox.count
+  expect(count).to eq expected_count.to_i
```

Comment:
If you only use `@receiver` inside this method you should stick to a local variable. If not then `@receiver` should be declared and set earlier in the scenario.